doc/modules/ROOT/pages/generate-jwt.adoc

```diff
@@ -102,6 +112,8 @@ Smallrye JWT supports the following properties which can be used to customize th
 [cols="<m,<m,<2",options="header"]
 |===
 |Property Name|Default|Description
-|smallrye.jwt.encrypt.key-location|none|Config property allows the location of a key to be specified which will be used to encrypt the claims or inner JWT.
-|smallrye.jwt.sign.key-location|none|Config property allows the location of a private key to be specified which will be used to sign the claims of a JWT.
+|smallrye.jwt.encrypt.key-location|none|Location of a public key which will be used to encrypt the claims or inner JWT when a no-argument encrypt() method is called.
```

Comment:
To be pedantic, this can be either a public key or a secret key. The encrypt()  API supports both

Reply:
@bobbyphilip Yeah, I forgot it will work if JWK format is used, I'll just update to a `signing key`, `encrypting key` for both properties...

Reply:
@bobbyphilip Or simply `a key` as per your original text